src/propcache/__init__.py

```diff
@@ -1,6 +1,30 @@
 """propcache: An accelerated property cache for Python classes."""
 
-__version__ = "1.0.0.dev0"
+from typing import TYPE_CHECKING, List
+
+__version__ = "0.2.0.dev0"
 
 # Imports have moved to `propcache.api` in 1.0.0+.
-__all__ = ()
+# This module is now a facade for the API.
+if TYPE_CHECKING:
+    from .api import cached_property, under_cached_property
+
+__all__ = ("cached_property", "under_cached_property")
```

Comment:
I just realized that this re-introduces the ability to get these via `from propcache import *`. Do we want this, or should this instead be a separate variable?

Reply:
I doubt anyone is doing that at this point so we can take that back out

Reply:
https://github.com/aio-libs/propcache/pull/32

Reply:
thanks!